Makefile

```diff
@@ -17,10 +17,10 @@
 CONTAINER_CMD?=docker
 
 RBD_IMAGE_NAME=$(if $(ENV_RBD_IMAGE_NAME),$(ENV_RBD_IMAGE_NAME),quay.io/cephcsi/rbdplugin)
-RBD_IMAGE_VERSION=$(if $(ENV_RBD_IMAGE_VERSION),$(ENV_RBD_IMAGE_VERSION),v1.0.0)
+RBD_IMAGE_VERSION=$(if $(ENV_RBD_IMAGE_VERSION),$(ENV_RBD_IMAGE_VERSION),v1.0.0-canary)
```

Comment:
(nit) should the defaults be something else other than canary? like "test" or "dev" or so.

Reply:
canary looks fine for me.

am not sure about the changes in  0.3 branch, people like @rootfs @gman0 who is having more idea on this can answer the question